### PR TITLE
Elide prelude components in the IR formatter.

### PR DESCRIPTION
**Pull request by jonmeow**
This is in particular to avoid churn from changes such as #4370. I think the import list can be helpful (particularly to understand what the library is aware of), but it's a different trade-off for the prelude package due to the implicit imports.

### CODE REVIEW

File: toolchain/sem_ir/formatter.cpp [hunk 1]
```diff
@@ -948,23 +961,10 @@ class FormatterImpl {
...
-    }
-    out_ << "//";
-    CARBON_CHECK(import_ir.library_id().is_valid());
-    if (import_ir.library_id() == LibraryNameId::Default) {
-      out_ << "default";
-    } else {
-      out_ << import_ir.string_literal_values().Get(
-          import_ir.library_id().AsStringLiteralValueId());
+      out_ << id;
     }
```
Comment (chandlerc) [inline toolchain/sem_ir/formatter.cpp:968]: Optional idea, I'm happy with either form though:

```suggestion
    out_ << (id.is_valid() ? GetImportIRLabel(id) : id);
```

**Review by chandlerc — APPROVED** [review]
Makes sense, especially for tests where the prelude is *so* noisy.

I wonder if we may want this to work the other way by default outside of file tests, but no reason to worry about that today. LG.

File: toolchain/sem_ir/formatter.cpp (hunk 1 above)
Comment (jonmeow) [reply toolchain/sem_ir/formatter.cpp:968]:
```
toolchain/sem_ir/formatter.cpp:964:28: error: incompatible operand types ('std::string' (aka 'basic_string<char>') and 'ImportIRId')
    out_ << (id.is_valid() ? GetImportIRLabel(id) : id);
                           ^ ~~~~~~~~~~~~~~~~~~~~   ~~
```

Comment (chandlerc) [reply toolchain/sem_ir/formatter.cpp:968]: :sigh: right, C++ makes that tricky. Oh well (and thanks).

**Pull request merged**